docusaurus/docs/Android/03-ui/06-guides/03-custom-message-composer.mdx

```diff
@@ -1,8 +1,15 @@
-# Creating a Custom Message Composer View
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-If the built-in [Message Composer View](../04-message-components/04-message-composer.mdx) and its available customization options don't fit your app's needs, you can create an Input View of your own.
+# Creating a Custom Message Composer
 
-Note that the UI Components Input View supports many advanced features that you'll otherwise have to implement yourself if you want to use them in your app:
+:::note
+You can find the full code from this guide on [GitHub](https://github.com/GetStream/stream-chat-android/tree/main/stream-chat-android-ui-guides/src/main/java/io/getstream/chat/android/guides/catalog/uicomponents/customcomposer). To check the final result, clone the repository and run the `stream-chat-android-ui-guides` module.
+:::
```

Comment:
Can we add more instructions for this, such as the image below?

<img width="752" alt="Screen Shot 2022-11-24 at 11 20 59 AM" src="https://user-images.githubusercontent.com/24237865/203679811-ade0bf25-0554-4bc0-8361-49f76c7b1d79.png">

Reply:
Yes, we need to improve this note everywhere. I created a task for that: https://github.com/GetStream/stream-chat-android/issues/4571